### PR TITLE
[ᚬrc/v0.15.z] test: fix integration test for #1169

### PR DESCRIPTION
**Pull request by quake**

### CODE REVIEW

**Comment by quake** [comment]
@nervos-bot-user try integration

File: test/src/specs/sync/sync_timeout.rs [hunk 1]
```diff
@@ -17,10 +17,10 @@ impl Spec for SyncTimeout {
...
         info!("Connect all nodes");
-        node0.connect(node1);
-        node0.connect(node2);
-        node0.connect(node3);
-        node0.connect(node4);
+        node1.connect(node0);
+        node2.connect(node0);
+        node3.connect(node0);
+        node4.connect(node0);
         net.waiting_for_sync(2);
```
Comment (keroro520) [inline test/src/specs/sync/sync_timeout.rs]: Expect the API reflects that who is inbound and who is outbound.
For example, `connect(inbound: Peer, outbound: Peer)`.

Comment (quake) [reply test/src/specs/sync/sync_timeout.rs]: I think that renaming parameter is clear enough:
```
pub fn connect(&self, outbound_peer: &Node)
```

let's do this refactoring in develop branch with different PR

**Comment by quake** [comment]
@nervos-bot-user try integration

**Comment by quake** [comment]
@nervos-bot-user try integration

**Pull request merged**